Package.swift

```diff
@@ -14,23 +14,22 @@ let excludeFiles: String = []
 let package = Package(
     name: "Web3swift",
     platforms: [
-        .macOS(.v10_12), .iOS(.v11)
+        .macOS(.v12), .iOS(.v15)
```

Comment:
Please downgrade this to `.v13` and `.v10_15` since it's the lowest versions that support [async/awaits](https://www.hackingwithswift.com/quick-start/concurrency/where-is-swift-concurrency-supported).

Reply:
@yaroslavyaroslav  are you sure? XCode demands 12.0.0
![Screen Shot 2022-04-24 at 10 23 09 AM](https://user-images.githubusercontent.com/38660775/164981149-b3988879-26cc-4624-90f2-9b1ca132b66e.png)